embedchain/embedchain.py

```diff
@@ -246,93 +246,7 @@ def add_local(
         )
         return self.add(source=source, data_type=data_type, metadata=metadata, config=config)
 
-    def load_and_embed(
```

Comment:
Can you please remove the `load_and_embed` function and also rename `load_and_embed_v2` to `load_and_embed` function?

Reply:
Sure

Reply:
Hey @deshraj , I made the mentioned changes.

Reply:
@deshraj The mentioned changes are made.